sources/engine/Xenko.Engine/Rendering/Compositing/ForwardRenderer.cs

```diff
@@ -172,13 +172,13 @@ protected override void InitializeCore()
                     }
                     vrSystem.PreferredScalings = preferredScalings;
 
-#if XENKO_GRAPHICS_API_VULKAN
-                    // no mirror support for vulkan
-                    vrSystem.RequireMirror = false;
-                    VRSettings.CopyMirror = false;
-#else
-                    vrSystem.RequireMirror =  VRSettings.CopyMirror;
-#endif
+                    if( GraphicsDevice.Platform == GraphicsPlatform.Vulkan ) {
```

Comment:
Please follow the global style (opening curly brace on new line)

Reply:
Also space after if but not inside parenthesis, i.e. `if (GraphicsDevice.Platform == GraphicsPlatform.Vulkan)`

Reply:
Never mind, fixed it directly with github online editor :)
Please try to follow those rules for other commits.